dio/lib/src/multipart_file.dart

```diff
@@ -140,4 +140,14 @@ class MultipartFile {
     _isFinalized = true;
     return _stream;
   }
+
+  /// Restore MultipartFile, returning a new instance of the same object. This is useful if your request failed and you wish
+  /// to sistematically retry it, for example a 401 error that can be solved by refreshing the token.
```

Comment:
Please update codes to follow the 80-word length format or semantic line breaks.
```suggestion
  /// Restore MultipartFile, returning a new instance of the same object.
  /// This is useful if your request failed and you wish to systematically retry it,
  /// such as an unauthorized exception can be solved by refreshing the token.
```

Reply:
I applied the line length rules to all the changes I made, thanks